pgbouncer/values.yaml

```diff
@@ -8,7 +8,14 @@ nodeAffinity: {} # optionally define nodeAffinity
 tolerations: [] # optionally define tolerations
 nodeSelector: {} # optionally define nodeSelector
 
-poolMode: session
```

Comment:
@angeloskyratzakos it's been a while - sorry!  it would be great to get your changes in.  could you make this last change - to re-add the poolMode line but commented out?